dunamis-editor/src/components/DynaformEditor/PalletePanel.vue

```diff
@@ -1,49 +1,21 @@
 <template>
   <v-expansion-panels v-model="panel" accordion multiple>
-    <v-expansion-panel
-      class="caption"
-      v-for="(group, i) in paletteElements"
-      :key="i"
-    >
+    <v-expansion-panel class="caption" v-for="(group, i) in paletteElements" :key="i">
```

Comment:
keep it in multiple lines, is most easy to read the code

Reply:
done